src/components/Form.jsx

```diff
@@ -1,57 +1,81 @@
 import { useState } from 'react';
+import { useDispatch } from 'react-redux';
 import { v4 as uuid } from 'uuid';
-import AddBookButton from './AddBookButton';
+import { postBookDetails, addBooks } from './redux/books/booksSlice';
 
-const INITIAL_FORM_STATE = {
-  author: '',
-  title: '',
-  id: '',
-};
+import '../styles/form.css';
+
+// const INITIAL_FORM_STATE = {
+//   item_Id: '',
+//   author: '',
+//   title: '',
+//   category: '',
+// };
```

Comment:
- [ ] To keep your code professional please remove all unused commented codes from your project.